src/widgets/courses/ui/courses.tsx

```diff
@@ -21,7 +20,7 @@ export const Courses = () => {
     filtered: false,
   };
 
-  const sortedCourses = getActualDataList(sortParams) as Course[];
+  const sortedCourses = getActualDataList(sortParams);
```

Comment:
I suggest to specify the type in this case. Yes, we can see the type here
![image](https://github.com/user-attachments/assets/4571f6a1-62aa-4090-9b82-bf426c92764e)

But if we additionally specify the type, we will be able to see what fields are in the Course type and it will be more convenient to access them without opening a file with the Course type
![image](https://github.com/user-attachments/assets/c5c3a897-fb1b-47b8-87e6-2e8c87ecb8c6)

Reply:
I'm used to relying on intellisense when working with fields, but if it would be more convenient for others - I've added an explicit type indication.